public-ui/src/app/views/quick-search/quick-search.component.ts

```diff
@@ -225,4 +250,42 @@ export class QuickSearchComponent implements OnInit, OnDestroy {
     return this.pmConceptGroups.filter(conceptgroup =>
       conceptgroup.groupName.toLowerCase().includes(searchString.toLowerCase())).length;
   }
+
+  public hoverOnTooltip() {
+    window['dataLayer'].push({
+      'event': 'tooltipsHover',
+      'category': 'Tooltips',
+      'action': 'Hover',
+      'label': 'Homepage Search Across Data',
+      'tooltipsHoverAction': 'Tooltip Homepage search across data'
+    });
+  }
+
+  public iconClickEvent(iconString: string) {
+    if (iconString === 'faq') {
+      window['dataLayer'].push({
+        'event': 'HelpEvent',
+        'category': 'Help',
+        'action': 'Click',
+        'label': 'FAQS'
```

Comment:
nit: can you not just use the iconString directly here?, this would avoid a lot of duplicate code here

Reply:
Done, thanks.